packages/documentation/astro.config.mjs

```diff
@@ -0,0 +1,161 @@
+import { defineConfig } from "astro/config";
```

Comment:
If we make the output dir be ` ./build`, we should be able to not have to change anything in Netlify and we'll get a nice preview link in the PR description to share